### ISSUE TITLE
Layering issue in week 4

### ISSUE THREAD

**Issue by Crappy-Productions** [ShadowMario/FNF-PsychEngine#13266]
### Describe your bug here.

Before you think about it: NO, I'M NOT TALKING ABOUT GF BEING MISPLACED.
As you can see here, BF is (supposed to) stand on top of the limo, while GF is in front of the limo. Yet somehow BF's foot is inbfront of GF's boom box.
![Screenshot_2023-09-11-15-40-42-149-edit_com android chrome](https://github.com/ShadowMario/FNF-PsychEngine/assets/98716926/60249865-7357-42e6-b883-7f08e657ad95)

So, I made up some 3D model images to represent it (I'm not good with 3D so don't judge me on how it looks).
![Screenshot_2023-09-11-16-18-03-203-edit_com android chrome](https://github.com/ShadowMario/FNF-PsychEngine/assets/98716926/d110c609-27ef-4b60-b002-02d4a029af5b)
![Screenshot_2023-09-11-16-18-21-333-edit_com android chrome](https://github.com/ShadowMario/FNF-PsychEngine/assets/98716926/ac445396-30cb-4726-bf7f-2e57eb5cdcad)

So basically the problem is BF's foot should not be in front of GF's boom box.

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

nope

**Comment by SrBagel** [comment]
I like to move gf to be right behind bf like this
![Screenshot 2023-09-15 160710](https://github.com/ShadowMario/FNF-PsychEngine/assets/128989598/bf217a21-652c-4531-851e-c76357c40839)
Not sure if this helps but its just a suggestion

**Comment by Crappy-Productions** [comment]
hmm that actiually look pretty nice, I'll try thay, thanks for your idea
